bulk-ingester/src/test/java/org/alfresco/hxi_connector/bulk_ingester/processor/mapper/TestNamespaceToPrefixMapper.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * #%L
+ * Alfresco HX Insight Connector
+ * %%
+ * Copyright (C) 2024 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+
+package org.alfresco.hxi_connector.bulk_ingester.processor.mapper;
+
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+@SuppressWarnings({"PMD.TestClassWithoutTestCases"})
+class TestNamespaceToPrefixMapper implements NamespacePrefixMapper
```

Comment:
Is this class necessary? Can't we just mock the mapping in the tests?

Reply:
it was much easier to create this test implementation, than trying to mock it as I had to implement just one method:
<img width="678" alt="image" src="https://github.com/Alfresco/hxinsight-connector/assets/62990104/d8f83e3f-2550-4eb1-9984-05485b839ce2">